tasks/expand_archive_posix.yml

```diff
@@ -130,9 +130,10 @@
   when: is_tar
 
 - name: "{{ archive.filename }}: Move out of base directory source"
-  ansible.posix.synchronize:
-    src: "{{ archive.base_destination_directory }}/{{ archive.move_and_delete_base_directory }}/"
-    dest: "{{ archive.base_destination_directory }}/"
+  ansible.builtin.command: >
```

Comment:
why not use ansible's buildtin copy? 
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/copy_module.html

Reply:
Updated.